pandasai/helpers/from_excel.py

```diff
@@ -0,0 +1,17 @@
+"""Functions to return a dataframe from an Excel file."""
```

Comment:
Note: PR #120 Uses a `datasource.py` module for the Google Sheets function. Whereas here a more specific module is used.